lib/measured/unit_system_builder.rb

```diff
@@ -13,8 +14,13 @@ def si_unit(unit_name, aliases: [], value: nil)
     nil
   end
 
+  def cache(cache_class, *args)
```

Comment:
Passing a class in here means anything that conforms to the `read` `write(table)` `exists?` API will seamlessly work here.

Reply:
🎉